resume-writing/resume-writing.md

```diff
@@ -1,109 +1,91 @@
 
 ### Guided Practice
 After reading each piece of resume advice, look at your resume again and edit accordingly.
 1. Download this [Entry-level IT developer resume template
-	](https://www.monster.com/career-advice/article/sample-resume-IT-developer-entry-level) from Monster.com.
-	
+    ](https://www.monster.com/career-advice/article/sample-resume-IT-developer-entry-level) from Monster.com.
 2. Read [this short reference about what to include in a basic resume](https://career.ucsd.edu/undergraduates/prepar-resume-covlet/writing-effective-resume.html).
-	
 3. Read this article:[Web Developer Resume: Sample & Complete Guide [+20 Examples]](https://zety.com/blog/web-developer-resume) (30 min read).
-	
 4. Read this short article on [How to show tech skills on your resume
 ](https://www.monster.com/career-advice/article/show-your-skills-on-your-it-resume).
-	
 5. Read this 20-min [Resume Writing Guide by JobScan](https://www.jobscan.co/resume-writing-guide).
-	
-6. Glance through [this list of Action Words](http://career.opcd.wfu.edu/files/2011/05/Action-Verbs-for-Resumes.pdf) and use them to replace any passive language in your resume. For example, replace "Was a scheduling manager at GAP" with "Managed and improved overall efficiancy of employee scheduling at GAP".
+6. Glance through [this list of Action Words](http://career.opcd.wfu.edu/files/2011/05/Action-Verbs-for-Resumes.pdf) and use them to replace any passive language in your resume. For example, replace "Was a scheduling manager at GAP" with "Managed and improved the overall efficiency of employee scheduling at GAP".
 
 ### Independent Practice
-
 1. Read this article on [How Subtle Class Cues Can Backfire on Your Resume](https://hbr.org/2016/12/research-how-subtle-class-cues-can-backfire-on-your-resume).
-
 2. Upload your resume and desired job description on https://www.jobscan.co/. Edit your resume or make a copy that incorporates all of the suggested missing keywords that apply to your background.
-
 3. Add your experience, and participation, and in some cases your resume document to your profile on these sites:
-	- LinkedIn
-	- Glassdoor
-	- AngelList
-	- Indeed
-	
+    - LinkedIn
+    - Glassdoor
+    - AngelList
+    - Indeed
+    
 ### Challenge
-
 1. Find a partner and review each other's updated resumes. Make any necessary edits.
 2. [Submit your completed resume to be reviewed on Monster.com](https://www.monster.com/resumes/post-resume2). You get one free resume review. They claim to get back to you within 2 days with your results.
 
 ### Supplemental Resources
 - [Tool: Build a Resume on Resume Genius](https://resumegenius.com/resume-formats)
+- [Entry-level IT developer resume template](https://www.monster.com/career-advice/article/sample-resume-IT-developer-entry-level) from Monster.com
```

Comment:
The idea is to move things to supplemental materials if they are NOT assigned in the lesson, but many of these are.  Can you double check and bring them back if they appear in any directions?